cmd/gcp-controller-manager/kubelet_readonly_csr_approver.go

```diff
@@ -105,8 +105,8 @@ func (approver *kubeletReadonlyCSRApprover) handle(ctx context.Context, csr *cap
 
 		switch {
 		case response.err != nil:
-			klog.Errorf("validating CSR %q: %v", csr.Name, err)
-			return fmt.Errorf("validating CSR %q: %v", csr.Name, err)
+			klog.Errorf("validating CSR %q: %v", csr.Name, response.err)
+			return fmt.Errorf("validating CSR %q: %v", csr.Name, response.err)
```

Comment:
perhaps a better error message: "csr %q: validator %q has error %v",csr.Name, validatorName, err

Reply:
perhaps we could even log/report both errors in this case
`klog.Errorf("validating CSR %q: %v failed; err: %v; response err: %v", csr, err, response.err)`